docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+provider, we want to separate them into their own namespace. For this demo, we
+will create a namespace called `app-project1-dev`, which we will use to group
+our Wordpress resources.
+
+* Define a `Namespace` in `app-project1-dev-namespace.yaml`:
+```yaml
+cat > app-project1-dev-namespace.yaml <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: app-project1-dev
+EOF
+```
+
+* Create the `Namespace`:
+```bash
+kubectl create -f app-project1-dev-namespace.yaml
+```
+
+* You should see the following output:
+```bash
+namespace/app-project1-dev created
+```
+
+#### Portable Resource Classes
+Portable resource classes are used to define a class of service in a single
+namespace for an abstract service type. We want to define our Azure
+`SQLServerClass` as the standard MySQL class of service in the namespace that
+our Wordpress resources will live in.
+
+* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace
+  `app-project1-dev`:
+```yaml
+cat > mysql-standard.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: app-project1-dev
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+EOF
+```
+
+* Create the `MySQLInstanceClass`:
+```bash
+kubectl apply -f mysql-standard.yaml
+```
+
+* You should see the following output:
+```bash
+mysqlinstanceclass.database.crossplane.io/mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-dev
+```
+*Output*
+```bash
+NAME             AGE
+mysql-standard   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this
+namespace to define more classes of service. For instance, if you created
+`mysql-azure-large` above, you may want to create a `MySQLInstanceClass` named
+`mysql-large` that references it. You may also choose to create MySQL resource
+classes for other non-Azure providers, and reference them for a class of service
+in the `app-project1-dev` namespace.
+
+You may specify *one* instance of a portable class kind as *default* in each
+namespace. This means that the portable resource class instance will be applied
+to claims that do not directly reference a portable class. If we wanted to make
+our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
+`app-project1-dev`, we could do so by adding a label:
+
+```yaml
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+```
+
+#### Resource Claims
+Resource claims are used to create external resources by referencing a class of
```

Comment:
What is a resource claim? As a new user, could I get a link to a document where I can read more about the concept?

Also, I feel this explanation would be a bit too dense for me as a new user. Something which goes a bit lower would be helpful, such as:

```
A resource claim is a Kubernetes object which tells Crossplane that a resource that lives outside of Kubernetes is desired.
```

The key part of it is explaining what the claim means, rather than just saying what it is used for.

Reply:
Hmm, this guide is likely not the first thing that a user will encounter. I think it is reasonable to assume that a user should have some concept of Crossplane concepts by the time they get here, and if they do not, they can always check out an earlier part of the docs. I worry that a simplified explanation is too abstract, and I usually desire for a guide to teach some about *how* things work, rather than just what to do. I can definitely see your point here about making docs more accessible though. I could certainly be misinterpreting the intended audience for this guide.